src/util/tr.js

```diff
@@ -466,6 +469,52 @@ const _tr = (msg, {
       }
     } else {
       renderFooterNotification(targetingMsgJson)
+
+      if (window.clevertap.hasOwnProperty('popupCallback') &&
+        typeof window.clevertap.popupCallback !== 'undefined' &&
+        typeof window.clevertap.popupCallback === 'function') {
+        const popupCallback = window.clevertap.popupCallback
+
+        const inaObj = {}
+        inaObj.msgContent = targetingMsgJson.msgContent
+        inaObj.msgId = targetingMsgJson.wzrk_id
+
+        var msgCTkv = []
+        for (var wzrkPrefixKey in targetingMsgJson) {
+          // ADD WZRK PREFIX KEY VALUE PAIRS
+          if (wzrkPrefixKey.startsWith('wzrk') && wzrkPrefixKey !== 'wzrk_id') {
+            const wzrkJson = { [wzrkPrefixKey]: targetingMsgJson[wzrkPrefixKey] }
+            msgCTkv.push(wzrkJson)
+          }
+        }
+
+        if (msgCTkv.length > 0) {
+          inaObj.msgCTkv = msgCTkv
+        }
+        if (targetingMsgJson.display.kv != null) {
+          inaObj.kv = targetingMsgJson.display.kv
+        }
+
+        // PUBLIC API TO RECORD CLICKED EVENT
+        window.clevertap.raisePopupNotificationClicked = (notificationData) => {
+          if (!notificationData || !notificationData.msgId) { return }
+
+          const eventData = {}
+          eventData.type = 'event'
+          eventData.evtName = 'Notification Clicked'
```

Comment:
@akashvercetti Can we define event name in constants class? Like [this](https://github.com/CleverTap/clevertap-ios-sdk/blob/master/CleverTapSDK/CTConstants.h#L26) .